docs/pages-for-subheaders/azure-marketplace-payg-integration.md

```diff
@@ -0,0 +1,227 @@
+
+#### Is this an annual commitment, will it auto-renew?
+
+By default, the Rancher Prime listing is billed on a monthly cycle, based on usage. Billing is on-going for as long as Rancher Prime is deployed.
+
+It is possible to set up an annual commitment via an Azure Private Offer, these will need to be reviewed and renewed at the end of the term, or the deployment will drop back to the default monthly billing cycle.
+
+### Technical
+
+#### Do I need a Kubernetes cluster running in Azure to install Rancher and be billed via the Azure Marketplace?
+
+Yes. To benefit from monthly billing via Azure, the primary Rancher cluster must be an Azure Kubernetes Service (AKS) cluster running in your Azure account. 
+
+#### On which Kubernetes distributions can the Rancher Prime Azure Marketplace listing be deployed?
+
+The Rancher Prime marketplace listing must be deployed on Azure Kubernetes Service (AKS). Downstream/managed clusters can run any supported Kubernetes platform: RKE, RKE2, EKS, GKE, vanilla Kubernetes, OpenShift, Mirantis Kubernetes Engine, etc. See Supported Platforms for more details.
+
+#### What is the deployment mechanism?
+
+The Rancher Prime marketplace listing is deployed using Azure’s CNAB (with Helm inside).
+
+#### What is the easiest way to get started?
+
+One of the easiest ways to get started is to deploy the Rancher Prime marketplace listing to an existing AKS cluster. Follow the instructions in the usage section of the listing. A Helm chart takes care of installation and billing setup.
+
+#### What is the minimum version of Rancher required to support Azure Marketplace billing?
+
+The minimum version supporting marketplace billing is Rancher 2.7.9.
+
+#### What version of Rancher is installed when using the marketplace listing?
+
+The Rancher Prime marketplace listing is tied to a specific version of Rancher, typically the latest version available at the time of the listing update. Check the listing for further information.
+
+#### I need a prior version of Rancher; can I still use the listing?
+
+No. There is no choice over the Rancher version when deploying using the Azure Marketplace listing. If a prior version of Rancher is required, this must be installed manually using the standard documentation. NOTE: Billing through the Azure Marketplace may not be supported with earlier versions.
+
+#### How often is the listing updated (including the version of Rancher, etc.)?
+
+The marketplace listing is tied to a specific version of Rancher, usually it is the latest version available at the time the listing was last updated. Typically, the marketplace listing is updated quarterly, or more frequently to address any new security issues. To update Rancher to a current version before the marketplace listing is updated, please see the product documentation.
```

Comment:
customers can only use versions that are supported by the listing, updating outside of the Azure specific update mechanism is likely to break billing, rendering them out of support

Reply:
@jeremy-moffitt removing this sentence: "To update Rancher to a current version before the marketplace listing is updated, please see the product documentation." 

b298c254cd1bb2f73cc92b1774d54b4f9d117c95